apps/avatax/turbo.json

```diff
@@ -23,10 +22,8 @@
         "SENTRY_PROJECT",
         "OTEL_ENABLED",
         "OTEL_SERVICE_NAME",
-        "VERCEL_GIT_COMMIT_SHA",
         "APP_LOG_LEVEL",
         "AVATAX_CLIENT_TIMEOUT",
-        "ENV",
```

Comment:
please test if turbo-ingore will not fail due to this, we must be sure our release model works.